templates/search/searchform.tpl

```diff
@@ -11,12 +11,9 @@
       <select name="type">
         <option value="Request">... requested username</option>
         <option value="email">... email address</option>
+	<option value="IP">... IP address</option>
         {if $currentUser->isAdmin() || $currentUser->isCheckuser()}
         <option value="IP">... IP address</option>
-        <option value="CIDR" disabled="disabled">... CIDR Range</option>{* not implemented yet! *}
-        {else}
-        <option value="IP" disabled="disabled">... IP address</option>
-        <option value="CIDR" disabled="disabled">... CIDR Range</option>
         {/if}
```

Comment:
Lines 15 through 17 are now completely redundant and should be removed entirely; as is, the code will show the IP Address option to admins and checkusers twice.